app/src/main/res/values/strings.xml

```diff
@@ -91,21 +91,24 @@
 
     <!-- App Backup and Restore State -->
 
+    <string name="backup_section_system">System Apps</string>
     <string name="backup_sms">SMS text messages</string>
     <string name="backup_settings">Device settings</string>
     <string name="backup_call_log">Call history</string>
     <string name="backup_contacts">Local contacts</string>
+    <string name="backup_section_user">User Apps</string>
```

Comment:
How about "Installed Apps" instead?

Reply:
would be fine with me as well. I used what the design had.

Reply:
"Installed" is just what's used everywhere in the OS I think, let's make the change and then this can be merged.

Reply:
done